ch_pipeline/core/dataquery.py

```diff
@@ -757,7 +766,7 @@ def next(
         return cont
 
 
-def finder_from_spec(spec, node_spoof=None):
+def finder_from_spec(spec, node_spoof=None, ntries=1):
```

Comment:
Could consider adding docstring entries for `node_spoof` and `ntries`, and also fix typo in docstring: `dspec` should be `spec`